libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -0,0 +1,437 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useRef, useState } from "react";
+import cx from "classnames";
+import { injectIntl, WrappedComponentProps } from "react-intl";
+import { IAnalyticalBackend, IAttributeElement } from "@gooddata/sdk-backend-spi";
+import {
+    filterAttributeElements,
+    filterObjRef,
+    IAttributeFilter,
+    idRef,
+    isAttributeElementsByRef,
+    newNegativeAttributeFilter,
+    newPositiveAttributeFilter,
+    ObjRef,
+} from "@gooddata/sdk-model";
+import Dropdown from "@gooddata/goodstrap/lib/Dropdown/Dropdown";
+import { AttributeDropdownBody } from "./AttributeDropdown/AttributeDropdownBody";
+import {
+    AttributeListItem,
+    EmptyListItem,
+    IElementQueryResultWithEmptyItems,
+    isNonEmptyListItem,
+} from "./AttributeDropdown/types";
+import debounce from "lodash/debounce";
+import isEmpty from "lodash/isEmpty";
+import isEqual from "lodash/isEqual";
+import noop from "lodash/noop";
+import { MAX_SELECTION_SIZE } from "./AttributeDropdown/AttributeDropdownList";
+import { mergeElementQueryResults } from "./AttributeDropdown/mergeElementQueryResults";
+import { IntlWrapper, withContexts } from "@gooddata/sdk-ui";
+import MediaQuery from "react-responsive";
+import { MediaQueries } from "../constants";
+import {
+    getAllTitleIntl,
+    getItemsTitles,
+    getNoneTitleIntl,
+    truncateTitle,
+} from "./utils/AttributeFilterUtils";
+
+/**
+ * @public
+ */
+export interface IAttributeFilterButtonOwnProps {
+    backend?: IAnalyticalBackend;
+    workspace?: string;
+    filter?: IAttributeFilter;
+    identifier?: string;
+    title?: string;
+    selectedItems?: Array<Partial<IAttributeElement>>;
+    onApply: (filter: IAttributeFilter, isInverted: boolean) => void;
+    onError: (error: any) => void;
+}
+
+/**
+ * @public
+ */
+export type IAttributeFilterButtonProps = IAttributeFilterButtonOwnProps & WrappedComponentProps;
+
+const DefaultFilterError: React.FC = injectIntl(({ intl }) => {
+    const text = intl.formatMessage({ id: "gs.filter.error" });
+    return <div className="gd-message error s-button-error">{text}</div>;
+});
+
+const DropdownBody: React.FC<{
+    isMobile?: boolean;
```

Comment:
So isMobile and closeDropdown are ignored here?

Reply:
The DropdownWrapper has no sense at all, left just AttributeDropdown a DropdownBody removed